src/Kafka/ManageKafkaPermissions/components/ManageAccessShortcut.tsx

```diff
@@ -0,0 +1,85 @@
+import { Flex, FlexItem, Popover, Title } from "@patternfly/react-core";
+import { Tbody, Td, Tr } from "@patternfly/react-table";
+import { useTranslation } from "react-i18next";
+import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon";
+import { ResourceTypeLabel } from "./ResourceTypeLabel";
+import { DisplayResourceName, PermissionOperationCell } from "./Cells";
+import { RemoveButton } from "../../../shared";
+import { ShortcutsTableHead } from "./ShortcutsTableHead";
+
+export type ManageAccessShortcutProps = {
+  onDelete: () => void;
+  multipleShorctutPermissions?: boolean;
+  instanceName: string;
+};
+export const ManageAccessShortcut: React.FC<ManageAccessShortcutProps> = ({
+  onDelete,
+  multipleShorctutPermissions = true,
+  instanceName,
+}) => {
+  const { t } = useTranslation(["manage-kafka-permissions"]);
+
+  return (
+    <>
+      {!multipleShorctutPermissions ? <ShortcutsTableHead /> : null}
```

Comment:
This header `ShortcutsTableHead` should not be in each shortcut component . it should be lift in parent component where we are integrating all these components.